src/main/java/com/amazon/opendistroforelasticsearch/ad/feature/SearchFeatureDao.java

```diff
@@ -121,12 +121,17 @@ public void getLatestDataTime(AnomalyDetector detector, ActionListener<Optional<
     }
 
     private Optional<Long> getLatestDataTime(SearchResponse searchResponse) {
-        return Optional
+
+        Optional<Long> x = Optional
             .ofNullable(searchResponse)
             .map(SearchResponse::getAggregations)
             .map(aggs -> aggs.asMap())
             .map(map -> (Max) map.get(AGG_NAME_MAX))
             .map(agg -> (long) agg.getValue());
+
+        System.out.println("windowDelay optional: " + x.toString());
+        System.out.println("windowDelay optional without .toString(): " + x);
+        return x;
```

Comment:
It looks like you add changes to just print out `LatestDataTime`. You may remove the changes if they are not needed anymore.

Reply:
fixed